modules/update-using-custom-machine-config-pools-inheritance.adoc

```diff
@@ -0,0 +1,206 @@
+// Module included in the following assemblies:
+//
+// * updating/updating_a_cluster/update-using-custom-machine-config-pools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="update-using-custom-machine-config-pools-mcp-inheritance_{context}"]
+= Managing machine configuration inheritance for a worker pool canary
+
+You can configure a machine config pool (MCP) canary to inherit any `MachineConfig` assigned to an existing MCP. 
+This is useful if, for example, you want to use a canary MCP to test as you update the nodes of an existing MCP, one at a time.
+
+
+.Prerequisites
+
+* You have created one or more custom machine config pools (MCP).
+
+.Procedure
+
+. Create a secondary MCP. 
+.. Save the following file as `machineConfigPool.yaml`.
```

Comment:
```suggestion
.. Save the following configuration file as `machineConfigPool.yaml`.
```
any specific directory?

Reply:
I don't think we need to specify a directory to save  it to. The user just saves it locally then applies it in the next step.